.buildkite/pipeline.yml

```diff
@@ -17,7 +17,7 @@ steps:
   branches: master
   env:
     BUILDKITE_CLEAN_CHECKOUT: true
-    TRAVIS_SCALA_VERSION: "2.12.12"
+    TRAVIS_SCALA_VERSION: "2.12.20"
```

Comment:
If this is still alive, it needs a 3.3.6 section.  Or maybe it works with the new 2.13.x and 3.x syntax.